mcs/class/System.Windows.Forms/Test/System.Windows.Forms/MdiFormHandleTest.cs

```diff
@@ -1098,16 +1098,10 @@ public void Show ()
 		{
 			SetUp ();
 			Assert.IsFalse (child1.IsHandleCreated, "A1");
-			child1.HandleCreated += new EventHandler (HandleCreated_WriteStackTrace);

```

Comment:
instead of removing the event handler we could set a variable that we assert below to ensure the event gets raised (assuming there's no such test already).

Reply:
I wasn't sure what was the purpose of the test. Will do it as you suggest.